cityofaustin/atd-moped
# Add sidebar with project details on selection of project components in map

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/9918

## Testing
**URL to test:** 

Local

**Steps to test:**
1. Go to the project list view and toggle the Map switch
2. You should see all project components load in the map and a collapsed sidebar on the left of the map
3. Expand the sidebar, see that there is a message that no components are selected, and then collapse it again
4. Hover components on the map and find a place where multiple of them overlap
5. Click and see that the sidebar opens and show a list of unique projects that have components overlapping in the place you clicked
6. You should see that all components associated with the projects in the list remain their status color and the rest are a muted gray
7. If you click a spot on the map where there are not components, the muted colors will regain their status colors
8. Try using advanced filters and click different areas of the map
9. Try a clicking links in the sidebar and you should be redirected to the project summary page of the project
10. Last, from a project summary page, click the link in the top left that says `< ALL PROJECTS`. You should be navigated back to the project list view with your previous filter. I created [another issue to capture the projects map toggle state in the search params](https://github.com/cityofaustin/atd-data-tech/issues/17601) so that it will navigate back to the map with the previous filters applied. Do y'all think that makes sense?

---
#### Ship list
